src/foundry/client/pixi/webgl/helpers/framebugger-snapshot.d.ts

```diff
@@ -0,0 +1,22 @@
+export {};
+
+declare global {
+  /**
+   * Provide the necessary methods to get a snapshot of the framebuffer into a render texture.
+   * Class meant to be used as a singleton.
+   * Created with the precious advices of dev7355608.
```

Comment:
Lol I wonder if this is here from before when they started working at Foundry.

Reply:
Seems likely, given that they were hired midway through v11 dev